README.md

```diff
@@ -1,223 +1,107 @@
-Data plane reliably delivers your event data. Control plane manages the configuration of your sources and destinations.
-This configuration can also be read from a file instead of from Control plane, if you don't want to use our hosted control plane.
+- **Client-side SDK Support**: RudderStack offers client-side SDKs for [JavaScript](https://docs.rudderstack.com/sdk-integration-guide/getting-started-with-javascript-sdk), [Android](https://docs.rudderstack.com/sdk-integration-guide/getting-started-with-android-sdk), [iOS](https://docs.rudderstack.com/sdk-integration-guide/getting-started-with-ios-sdk), and [Unity](https://docs.rudderstack.com/sdk-integration-guide/getting-started-with-unity-sdk)
 
-Config-generator provides the UI to manage the source and destination configurations without needing to signup, etc.
-All the source and destination configuration stays on your local storage. You can export/import config to a JSON file.
+- **User-specified Transformation**: RudderStack gives you the power to filter or transform your events before sending them to the desired destinations
 
-## Setup
+# Key Features
 
-1. `cd utils/config-gen`
-2. `npm install`
-3. `npm start`
+- **Segment API Compatible**: RudderStack is Segment API and library compatible. So you don't need to change your app if you are using Segment.
 
-RudderStack config generator starts on the default port i.e., http://localhost:3000.
-On a successful setup, you should see the following
+- **Processing Flexibility**: With RudderStack's powerful JavaScript-based event transformation framework, you can enhance or transform your event data by combining it with your other internal data. Furthermore, since RudderStack runs inside your cloud or on-premise environment, you can easily access your production data to join with the event data.
 
-![image](https://blobscdn.gitbook.com/v0/b/gitbook-28427.appspot.com/o/assets%2F-Lq586FOQtfjJPKbd01W%2F-M0LidOVklHOkLYEulS4%2F-M0M3fzyrb-UHiNBG7j0%2FScreenshot%202020-02-18%20at%2012.20.57%20PM.png?alt=media&token=a3f24ad8-fe72-4fed-8953-8e4c790f6cfd)
+- **Unlimited Events**: Event volume-based pricing of most of the commercial systems is broken. With RudderStack, you are be able to collect as much data as possible without worrying about overrunning event budgets.
 
-## Export workspace config
+- **Stand-alone System**: RudderStack runs as a single Go binary with the only dependency being on the PostgreSQL database. There is no other dependency required to use RudderStack.
 
-After adding the required sources and destinations, export your workspace config. This workspace-config is required by the RudderStack Server.
-To learn more about adding sources and destinations in RudderStack, refer [Adding a Source and Destination in RudderStack](https://docs.rudderstack.com/getting-started/adding-source-and-destination-rudderstack)
+- **High Performance**: On a single m4.2xlarge, RudderStack can process 3000 events/second.
```

Comment:
Add 1 more section: *Platform independent* RudderStack is Kubernetes native and can be run in any Kubernetes cluster. RudderStack is cloud-agnostic and can be run on stand-alone machines in all cloud platforms.

@ameypv-rudder Add a link to the following repo for the Kubernetes setup.
https://github.com/rudderlabs/rudderstack-helm